src/CommunityVoices/Public/js/tag-collection.js

```diff
@@ -0,0 +1,49 @@
+// form is the form that we are submitting (an edit form)
+// whatUpdated is a string of what we will say we updated
+function submitEdit (form, whatUpdated) {
+    var data = $(form).serializeArray();
+
+    $('#alert').addClass('alert-success').removeClass('d-none alert-danger');
+    $('#alert-content').text('Updating...');
+
+    $.ajax({
+      url : $(form).attr('action') || window.location.pathname,
+      type: $(form).attr('method') || "POST",
+      data: $.param(data),
+      success: function (data) {
+          $('#alert-content').text('Updated ' + whatUpdated);
+      },
+      error: function (data) {
+          $('#alert').addClass('alert-danger').removeClass('d-none alert-success');
+          $('#alert-content').text('Failed to update ' + whatUpdated);
+      }
+    });
+}
+$('.edit-form').on('submit', function(e) {
+  e.preventDefault();
+
+  submitEdit(this, this.elements.label.value);
+});
+
+$('.delete-form').on('submit', function(e) {
+  e.preventDefault();
+  var form = $(this);
+  var action = form.attr('action');
+  $('#alert').addClass('alert-danger').removeClass('d-none alert-success');
+  $('#alert-content').html('<strong>Warning:</strong> Deleting tags will affect related ');
+  var btn = document.createElement("button");
```

Comment:
I wonder if JavaScript's `confirm` would make more sense - we use it elsewhere and it forces a decision to be made.  This is nifty, though

Reply:
You're definitely welcome to disagree here and continue to use this way of doing it.